code/datums/flock/flock.dm

```diff
@@ -651,12 +659,14 @@
 	if(lat)
 		qdel(lat)
 		T.ReplaceWith("/turf/simulated/floor/feather", 0)
+		APPLY_ATOM_PROPERTY(T, PROP_ATOM_FLOCK_THING, "flock_convert_turf")
```

Comment:
Wouldn't it make more sense to do this in the `New` of the flock types, or am I misunderstanding what this atom property is for?

Reply:
It can be done in either location safely, but most things that the flock creates happen here.